frontend/src/config/oidcConfig.ts

```diff
@@ -0,0 +1,6 @@
+export default {
+    authority: "http://localhost:2000/auth/realms/master",
```

Comment:
Kein eigener Realm? Vielleicht hatte uich Dich da falsch verstanden, aber das wäre schon besser. Außerdem gehören die Werte eher in eine .env Datei.

Reply:
Oder andere Art von Konfiguration.